lib/V8/v8-shell.cpp

```diff
@@ -266,9 +267,15 @@ static void JS_ProcessJsonFile(v8::FunctionCallbackInfo<v8::Value> const& args)
   // extract the callback
   v8::Handle<v8::Function> cb = v8::Handle<v8::Function>::Cast(args[1]);
 
-  // read and convert
-  std::string line;
+// read and convert
+std::string line;
+#ifndef _MSC_VER
   std::ifstream file(*filename);
+#else
+  std::ifstream file;
```

Comment:
imho we should use something from basics to do this. We have a porting layer, we should not distribute #if wintendo all over the place.

Reply:
We use the steams only in a few places and as you see a function from basics was used...

Reply:
Again more macros and legacy c functions are not the solution. Lets wait for c++17 support. Then we should refactor the filesystem related functions with std::filesystem. As I said in the retro: "Can we please distinguish between fixes and beautification. This PR is about testing and fixing the code so that one of the tests works. What is next for this PR? Maybe remove the 4 or 5 `#ifdef`s in v8-utils.cpp? How about doing internal API improvement in a dedicated PR? Must we always mix stuff up?